docs/system-admin-guide/authentication/saml/README.md

```diff
@@ -61,11 +61,13 @@ saml:
   assertion_consumer_service_url: "https://<YOUR OPENPROJECT HOSTNAME>/auth/saml/callback"
   issuer: "https://<YOUR OPENPROJECT HOSTNAME>"
 
-  # IF your SSL certificate on your SSO is not trusted on this machine, you need to add it here
+  # IF your SSL certificate on your SSO is not trusted on this machine, you need to add it here in ONE line
+  ### one liner to generate certificate in ONE line
+  ### awk 'NF {sub(/\r/, ""); printf "%s\\n",$0;}' <yourcert.pem>
```

Comment:
:bow:

Reply:
What does this emoji actually stand for? it seems to have no information to me... at least i cannot combine to anything else than a person with some green shirt